ghtt/assignment.py

```diff
@@ -318,6 +318,68 @@ def create_repos(ctx, source, yes, students=None, groups=None):
         g_repo.edit(description=repo.comment)
 
 
+@assignment.command()
+@click.pass_context
+@click.option(
+    '--students',
+    help='Comma-separated list of usernames. Defaults to all students.')
+@click.option(
+    '--groups',
+    help='Comma-separated list of group names. Defaults to all groups.')
+@click.option(
+    '--destroy-data',
+    help='Confirm that you want to use this command that can destroy data.', is_flag=True)
+def delete_repos(ctx, students=None, groups=None, destroy_data=False):
+    """Delete student repositories in the organization specified by the url.
+
+    WARNING: this is obviously a dangerous operation!"""
+
+    click.secho("*** This command will delete repositories! ***", fg="red")
+    click.secho("*** It will destroy data irrecoverably! ***", fg="red")
+    click.secho("\nLifesaver: Consider using the command \"ghtt assignment rename-repo\" to rename repos "
+                "instead of deleting them.\n", fg="green")
+
+    if not destroy_data:
+        click.secho("Add the command line option --destroy-data to enable the delete-repos command.")
+        exit(1)
+
+    if not ghtt.config.get("enable-repo-delete", False):
+        click.secho("Add the line \"enable-repo-delete: True\" to your ghtt config to enable the delete-repos command.")
+        exit(1)
+
+    yes = False  # --yes has been disabled. Better always be safe with delete.
+
+    if students:
+        students = [s.strip() for s in students.split(",")]
+    if groups:
+        groups = [gr.strip() for gr in groups.split(",")]
+
+    click.secho("# Deleting (!!!) student repositories..", fg="red")
```

Comment:
```suggestion
    click.secho("# WARNING! Deleting student repositories. This will permanently remove all code and history of these repositories! For a less destructive way to remove a repository, see 'ghtt assignment rename-repo'.", fg="red")
```